OTAnalytics/plugin_ui/customtkinter_gui/toplevel_flows.py

```diff
@@ -111,10 +162,16 @@ def _is_float_above_zero(self, entry_value: Any) -> bool:
         return float_value >= 0
 
     def _sections_are_valid(self) -> bool:
```

Comment:
As I read this method again now, I have the feeling, that this is too much responsibility for a "dumb" UI. Should we move this functionality into the application layer? (Maybe same applies to some other methods in the gui code - just schedule a refactoring for now?)

Reply:
We can move it into the view model or application code. But I would move it in a future clean up refactoring.